action/protocol/producer/admin.go

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package producer
+
+import (
+	"bytes"
+	"context"
+	"math/big"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/action/protocol"
+	"github.com/iotexproject/iotex-core/action/protocol/producer/producerpb"
+	"github.com/iotexproject/iotex-core/address"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// admin stores the admin data of the block producer protocol
```

Comment:
nit: "block producer protocol" is not accurate. "rewarding protocol" might be better?

Reply:
ack